src/steamship/data/package/package_instance.py

```diff
@@ -109,3 +113,37 @@ def invoke(
 
     def full_url_for(self, path: str):
         return f"{self.invocation_url}{path}"
+
+    def refresh_init_status(self):
+        new_self = PackageInstance.get(self.client, handle=self.handle)
+        self.init_status = new_self.init_status
+
+    def wait_for_init(
+        self,
+        max_timeout_s: float = 180,
+        retry_delay_s: float = 1,
+    ):
+        """Polls and blocks until the init has succeeded or failed (or timeout reached).
+
+        Parameters
+        ----------
+        max_timeout_s : int
+            Max timeout in seconds. Default: 180s. After this timeout, an exception will be thrown.
+        retry_delay_s : float
+            Delay between status checks. Default: 1s.
+        """
+        t0 = time.perf_counter()
+        refresh_count = 0
```

Comment:
Don't think you're doing something with the variable aside from incrementing it. You can probably drop it since you're already using time.perf_counter()

Reply:
Quite right, thanks!